pkg/volume/csi/csi_plugin.go

```diff
@@ -207,7 +207,8 @@ func (p *csiPlugin) Init(host volume.VolumeHost) error {
 	if utilfeature.DefaultFeatureGate.Enabled(features.CSIDriverRegistry) {
 		csiClient := host.GetKubeClient()
 		if csiClient == nil {
-			return errors.New("unable to get Kubernetes client")
+			klog.Warning(log("kubeclient not set, assuming standalone kubelet"))
```

Comment:
i get we want to skip informer init. but do we also want to skip `nodeinfomanager` init? Although we wont be able to use the nodeinfomanager since it just does apiserver stuff anyway, will `nim== nil` cause problems elsewhere?

Reply:
changed to just skip informer initialization. and verified that the rest of plugin code checks for nil informer